deploy/manifests/base/storetheindex/ingest-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: ingest
+  labels:
+    app: ingest
+spec:
+  ports:
+    - name: ingest
+      port: 3001
+      targetPort: ingest
+  selector:
+    app: indexer
+  type: LoadBalancer
```

Comment:
If we do not expose the admin port, then this is certainly not needed.  Even if we do expose the admin port, do we want that going through the load balancer since it is only used to handle a few specific commands?

Reply:
Comment was referring to admin port above.  NM if not exposed.